### PR TITLE
Ruby SDK Updates

### PR DESCRIPTION
**Pull request by JamieSinn**

### CODE REVIEW

File: docs/sdk/server-side-sdks/ruby.md [hunk 1]
```diff
@@ -53,12 +49,9 @@ end
 require 'devcycle-server-sdk'
 
 # Setup authorization
-DevCycle.configure do |config|
-  # Configure API key authorization
-  config.api_key['bearerAuth'] = '<DVC_SERVER_SDK_KEY>'
-end
+options = DevCycle::DVCOptions.new(enable_cloud_bucketing: false, event_flush_interval_ms: 1000, config_polling_interval_ms: 1000)
+dvc_client = DevCycle::DVCClient.new("dvc_server_token_hash", options, true)
```
[the remark this answers is not in the document]
Comment (JamieSinn) [reply docs/sdk/server-side-sdks/ruby.md:53]: sdkKey is the parameter name. The string is the example token format.

[the remark this answers is not in the document]
Comment (VicV) [reply docs/sdk/server-side-sdks/ruby.md:53]: seems weird to call it something different in the example still though :p

[the remark this answers is not in the document]
Comment (JamieSinn) [reply docs/sdk/server-side-sdks/ruby.md:53]: We're not? That's the parameter value. 
I can prefix the value with the param name to make it clear if you want.

**Comment by VicV** [comment]
we like this

**Pull request merged**